eng/Versions.props

```diff
@@ -46,5 +46,6 @@
     <MicrosoftEntityFrameworkCoreSqlServerPackageVersion>8.0.2</MicrosoftEntityFrameworkCoreSqlServerPackageVersion>
     <MicrosoftEntityFrameworkCoreToolsPackageVersion>8.0.2</MicrosoftEntityFrameworkCoreToolsPackageVersion>
     <MicrosoftNETRuntimeWorkloadTestingInternalVersion>9.0.0-preview.3.24151.4</MicrosoftNETRuntimeWorkloadTestingInternalVersion>
+    <TestcontainersPackageVersion>3.7.0</TestcontainersPackageVersion>
```

Comment:
Let's leave `eng\Versions.props` only have the versions that are managed by darc in it. Can you move this shared property to the top of `Directory.Packages.props`? I dislike having to look in multiple places. And when we get dependabot working again, I'm not sure it knows how to update `eng\Versions.props` files.